index.html

```diff
@@ -9,11 +9,27 @@
     <title>Document</title>
 </head>
 <body>
-    <h1 id="header"></h1>
+    <!-- *-------------------  This is Header Site  ----------------------------->
+    <header id="header">
+        <nav id="navbar">
 
-    <h2 id="subheader"></h2>
+        </nav>
+        <div id="banner">
 
-    <h3 id="subheader_2"></h3>
+        </div>
+    </header>
+    <!-- *-------------------  This is Main Section Site  ----------------------------->
+    <main id="main">
+        <div id="content">
```

Comment:
In this section, you could put both content and sidebar inside the same body tag instead of inside a main tag, but this is also good.